### PR TITLE
chore: updated react-scripts and relevant lib

### PR DESCRIPTION
**Pull request by justussoh**
This PR:
- Updates react-script to 4.x
- Fixes broken deps after update

### CODE REVIEW

File: integration/integration-end-to-end.spec.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ const EblNumberField = Selector("input#root_blNumber");
 const previousDocumentButton = Selector("[data-testid='previous-document-button']");
 const nextDocumentButton = Selector("[data-testid='next-document-button']");
 const fileNameField = Selector("[data-testid='file-name-input']");
-const goBack = ClientFunction(() => window.history.back());
+const goBack = ClientFunction(() => window.close());
```
Comment (Nebulis) [inline integration/integration-end-to-end.spec.ts:28]: go back = close ?

File: src/components/DynamicFormContainer/DocumentPreview/DocumentPreview.stories.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import React from "react";
+import React, { FunctionComponent } from "react";
 import { Container } from "../../Container";
 import { DocumentPreview } from "./DocumentPreview";
 import SampleCnm from "../../../test/fixtures/unwrapped-cnm.json";
-import { Document } from "@govtechsg/decentralized-renderer-react-components";
+import { v2 } from "@govtechsg/decentralized-renderer-react-components";
```
Comment (Nebulis) [inline src/components/DynamicFormContainer/DocumentPreview/DocumentPreview.stories.tsx:5]:
```suggestion
import { v2 } from "@govtechsg/open-attestation";
```

Comment (Nebulis) [reply src/components/DynamicFormContainer/DocumentPreview/DocumentPreview.stories.tsx:5]: Everywhere

File: integration/integration-end-to-end.spec.ts (hunk 1 above)
Comment (justussoh) [reply integration/integration-end-to-end.spec.ts:28]: This function opens a new tab, by right the function we should be using is `close` else test fails.

Comment (justussoh) [reply integration/integration-end-to-end.spec.ts:28]: me will rename it to close

**Review by Nebulis — APPROVED** [review]

**Pull request merged**